mail_full_expand/static/src/components/message/mail_full_expand.js

```diff
@@ -0,0 +1,29 @@
+/* Copyright 2014-2015 Grupo ESOC <http://www.grupoesoc.es>
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+ */
+odoo.define(
+    "mail_full_expand/static/src/components/message/mail_full_expand.js",
+    function (require) {
+        "use strict";
+
+        const components = {
+            Message: require("mail/static/src/components/message/message.js"),
+        };
+        const {patch} = require("web.utils");
+
+        patch(
+            components.Message,
+            "mail_full_expand/static/src/components/message/mail_full_expand.js",
```

Comment:
I'm not sure if this would be the best patch name. [It looks like `module_name.key` is recommended](https://github.com/odoo/odoo/blob/bf350c3015f86357e609a6302b48e7c22f2c0b47/addons/web/static/src/js/core/patch_mixin.js#L19) instead.

```suggestion
            "mail_full_expand.main",
```